lib/model/store.dart

```diff
@@ -570,7 +571,21 @@ class LiveGlobalStore extends GlobalStore {
 
   @override
   Future<Account> doInsertAccount(AccountsCompanion data) async {
-    final accountId = await _db.createAccount(data); // TODO(log): db errors
+    final int accountId;
+    try {
+      accountId = await _db.createAccount(data); // TODO(log): db errors
+    } catch (e) {
+      switch (e) {
+        case DriftRemoteException(
+            remoteCause: SqliteException(
+              extendedResultCode: 2067 // SQLITE_CONSTRAINT_UNIQUE
```

Comment:
Interesting. What reference did you use to determine that this was the numeric value to expect for this error?

I think I'd want this to be a named constant somewhere, and then that somewhere can have a comment linking to such a reference.

Ideally `package:sqlite3` would provide all these codes as named constants. Failing that, we can put this one in our `lib/model/database.dart`, and add more codes next to it as we need them. It can look like
```dart
abstract class SqliteResultCode {
  /// The SQLite result code `SQLITE_CONSTRAINT_UNIQUE`.
  const ConstraintUnique = 2067;
}
```
with some dartdoc on the class that includes a link to the relevant reference.

Reply:
Hm good point, I imported sqlite3 and used the exported error code constant instead.